src/components/Accordion.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import vectorMinus from '../assets/vectorShow'
+import vectorMore from '../assets/vectorHidden'
+
+const AccordionContainer = styled.div`
+  display: flex;
+`
+
+const Title = styled.h1``
+
+const Content = styled.p``
+const Accordion = props => {
```

Comment:
```suggestion
const Accordion = ({ content, title }) =>
  <AccordionContainer>
    <Title> {title}</Title>
    <Content> {content}</Content>
  </AccordionContainer>
```
Da pra simplificar usando `destructuring` pra receber as props e quando você estiver retornando um bloco só de `markup` não precisa das `{}`